Dispatcher/Constants/AppConstants.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  AppConstants.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 30/04/2023.
+//
+
+import Foundation
+
+struct AppConstants {
```

Comment:
I would change the file and the struct to "CellConstants" and have only cell-related stuff in here

Reply:
Also I would suggest having two separate structs inside this file - struct TableCellsIdentifier & struct NibNames